### ISSUE TITLE
Detection of executables named "0"

### ISSUE THREAD

**Issue by eserte** [plicease/File-Which#7]
A hypothetical executable named "0" is not detected by `which`:

```
$ touch 0 1; chmod 755 0 1
$ env PATH=$(pwd):$PATH pistachio-perl -MFile::Which -E 'say which("0"); say which("1")'

/tmp/1
```

Probably an additional `defined` in https://metacpan.org/source/PLICEASE/File-Which-1.19/lib/File/Which.pm#L46 would fix this.

**Comment by plicease** [comment]
Thank you for the report.  I believe this to be fixed in 1.20.  Please reopen if you disagree.

**Comment by eserte** [comment]
Thanks, looks good!
